entrypoint.sh

```diff
@@ -51,27 +51,51 @@ case ${RUNNER_SCOPE} in
     ;;
 esac
 
-if [[ -n "${ACCESS_TOKEN}" ]]; then
-  _TOKEN=$(bash /token.sh)
-  RUNNER_TOKEN=$(echo "${_TOKEN}" | jq -r .token)
+# If the variable is not set, set it with the default value
+if [ -z "${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}" ]; then
```

Comment:
this also makes it work by default vs opt-in

Reply:
See the thread with the same topic above